RELEASE.md

```diff
@@ -0,0 +1,52 @@
+Release type: minor
+
+This release adds `strawberry.input_mutation` as a way to make it easier to create
+mutations that receive a single input type called `input`, while still being able
+to define the arguments of that input in the resolver itself.
+
+The following example:
+
+```python
+@strawberry.type
+class Fruit:
+    id: strawberry.ID
+    name: str
+    weight: float
+
+
+@strawberry.type
+class Mutation:
+    @relay.input_mutation
```

Comment:
Great feature!
Should this be `@strawberry.input_mutation`?

Reply:
Nice catch. I ported this from the relay PR and forgot to adjust after ctrl+c/ctrl+v